v1.0.0

matthewtiscareno opened the pull request:
<!--
  Title ^^^^ above ^^^^ should follow good commit message best practices wherever possible.

  A properly formed git commit subject line should always be able to complete the following sentence:

    If applied, this commit will <your subject line here>
-->

## Summary
<!-- Brief summary of changes if not sufficiently described by commit messages. -->
Release dictionary with v1.0.0

## Test Data and/or Report
<!--
  One of the following should be included here:
    * Reference to regression test included in code (preferred wherever reasonable)
    * Attach test data here + outputs of tests
-->
Enter info here

## Related Issues
<!--
  Reference related issues here and use `Fixes` or `Resolves` for closing issues, for example:
  * Refs pds-data-dictionaries/PDS4-LDD-Issue-Repo#13 will include a link between this PR and that issue
  * Resolves pds-data-dictionaries/PDS4-LDD-Issue-Repo#13 will actually close the ticket once this PR is merged
-->
Enter related issues here

mit3ch commented:
This looks ready to build. There is one minor set of edits that you should consider fixing before the public release. Before each of the classes, there is a descriptive comment (e.g., <!--      Observing program attributes     -->) in which "attributes" should be replaced with "class".

mace-space reviewed (approved):
Subject to Mitch's (minor) edits, this is approved

matthewtiscareno commented:
> There is one minor set of edits that you should consider fixing before the public release. Before each of the classes, there is a descriptive comment (e.g., "-- Observing program attributes --") in which "attributes" should be replaced with "class".

Done, I think.

markshowalter commented on src/PDS4_HST_IngestLDD.xml:
Lines 194-195 fix wording:
`For HST instruments that do not have multiple channels, such as WFPC2, this
attribute has the same value as the instrument_id attribute.`

Starting at line 555, there's no clear explanation of how `observation_type` gets a value of "TIME-SERIES", and there were also a few other issues. I recommend this revision:
```
            The type of the observation, one of IMAGING, SPECTROSCOPIC, or TIME-SERIES.
            IMAGING indicates that a 2-D image with two spatial axes was obtained.
            SPECTROSCOPIC indicates that one of the axes of the data array is related to
            wavelength. TIME-SERIES indicates that one of the axes is sampling time.

            For many HST data files, this is the value of FITS keyword OBSTYPE. Where
            OBSTYPE does not appear, this value depends on other information.
            - All ACS, NICMOS, WFC3, WFPC, and WFPC2 observations are IMAGING.
            - All COS, FOS, and GHRS observations are SPECTROSCOPIC.
            - All HSP observations are TIME-SERIES.
            - For FOC, the observation_type depends on the value of FITS keyword OPMODE:
              IMAGING if OPMODE is "ACQ" or "IMAGE"; TIME-SERIES of OPMODE is "OCC";
              SPECTROSCOPIC if OPMODE is "SPEC".

            Note that, for ACS/HRC, the FITS keyword OBSTYPE can have a value of
            CORONAGRAPHIC. However, the value of this attribute is IMAGING; the
            coronagraph_flag attribute indicates whether the coronagraph was used.
```

markshowalter reviewed (changes requested):
My comments are just revisions of two definitions. Somebody should look them over but then they should be suitable for cut-and-paste into the document. I mentioned on Slack that `moving_target_flag` should probably not be nillable. Everything else looks good.

matthewtiscareno commented:
Per @markshowalter's review, I have done the following: 
- I added his suggested text for `channel_id` and `observation_type`.  For the latter, I changed his first comma into a colon, and I also changed the last sentence so that now reads as follows: 
`In such cases, the value of this attribute is IMAGING and the coronagraph_flag attribute indicates whether the coronagraph was used.`
- I added a corresponding sentence to `coronagraph_flag` explaining the same concept.
- I removed nillability from `moving_target_flag`
- I added sentences to the definitions of `hst_quality_id` and `hst_quality_comment`, as we're doing for other nillable attributes, explaining when they might be nil and which value of `nilReason` should be used.

You should be able to see all of this in the diff.  Please review before we proceed with the merge.
